WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/BlockEditorPage.java

```diff
@@ -42,13 +40,12 @@ public BlockEditorPage waitForTitleDisplayed() {
     }
 
     public BlockEditorPage enterTitle(String postTitle) {
-        titleField.perform(typeText(postTitle), ViewActions.closeSoftKeyboard());
+        populateTextField(titleField, postTitle);
```

Comment:
> Even with this, the execution became faster, because populateTextField uses replaceText and not typeText.

1) Question (❓): I wonder if the author of this test explicitly wanted `typeText` instead of `replaceText`? The former has the effect of actually typing the text, while the latter will just replace the text. This then might not be what the user would do, thus the test would differ from the expected scenario that the author wanted in the first place. Wdyt? 🤔 
2) Question (❓): I am seeing the `populateTextField(...)` utility method being used by multiple tests, not only the block editor related ones (like login, signup, help and support, etc). Are we sure the `clickOn(element)` addition isn't somehow negatively affecting those other tests? 🤔

Reply:
Both are great questions @ParaskP7, and they relate to the topic of balance between test stability and preserving the user aspect.

> I wonder if the author of this test explicitly wanted `typeText` instead of `replaceText`?

I think we can go forward and ask @tiagomar about his thoughts on this. You're absolutely correct that directly setting the text into the input is not something an ordinary user would do by a power of thought. Personally, I try to avoid `setText` / `replaceText` sort of thing in tests when possible, and one of the reasons why I went here for it is `typeText` slowness. Entering a lengthy paragraph of text takes some time on FTL.

I partly addressed this point in ae8359a276e4d37b059015f0183d81d9a52bf472: I'm now using 'replaceText` for paragraphs only, and still using `typeText` for post header. This way, we still do some "real" typing when we can.

> Are we sure the `clickOn(element)` addition isn't somehow negatively affecting those other tests? 🤔

I had the same thought, and initially it seemed to me that seeing all tests green was enough. However, I can't say that I followed every miniscule aspect of execution to say that it made no impact on the rest of the tests. It might be that we just don't have enough assertions in tests to notice this.

Anyway, since this was pretty easy to address, I relocated `clickOn()` to two methods where it was needed, via b9263072c48e4ac4c2e03ab04534bc5830024696.

Thank you!

Reply:
Wow, thank you so much for addressing this questions with your actions! ❤️ 

> I partly addressed this point in https://github.com/wordpress-mobile/WordPress-Android/commit/ae8359a276e4d37b059015f0183d81d9a52bf472: I'm now using 'replaceTextfor paragraphs only, and still usingtypeText` for post header. This way, we still do some "real" typing when we can.

Love it! 🥇 

> Anyway, since this was pretty easy to address, I relocated clickOn() to two methods where it was needed, via https://github.com/wordpress-mobile/WordPress-Android/commit/b9263072c48e4ac4c2e03ab04534bc5830024696.

Nicely done! 🥇 

I reviewed both changes and they LGTM! 🙇 ✅